modal.js

```diff
@@ -0,0 +1,134 @@
+    livelink: live,
+    sourcelink: source,
+  },
+
+];
+
+const proj0 = {
+  title: 'Multi Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  mainimage: 'Images/modalimg.png',
+  technologies: ['HTML', 'Bootstrap', 'Ruby'],
+  livelink: live,
+  sourcelink: source,
+};
+
+for (let i = 0; i < projectsarr.length; i += 1) {
+  const index = i;
+  const project = projectsarr[i];
+  let technologieslang = '';
+  for (let j = 0; j < project.technologies.length; j += 1) {
+    const tech = project.technologies[j];
+
+    technologieslang += `<li class="lang-li_2">${tech}</li>`;
+  }
+  document.querySelector('#other-proj-contain').innerHTML
+  += ` 
+  <div class="bk-img-proj">
+      <h2 class="bk-img-title">${project.title}</h2>
+      <p class="bk-img-p">${project.description}</p>
+      <ul class="lang-contain_2">
+        ${technologieslang}
+       </ul>
+      <button onclick='projectdetails(${index})' class="st-see-btn_2" type="button">See Project</button>
+  </div>
+  `;
+}
+
+function projectdetails(index) {
+  const dataobj = projectsarr[index];
+  const popuptitletext = document.querySelector('#popuptitle');
+  const popupimg = document.querySelector('.popimg');
+  const popupdescription = document.querySelector('.popupdescription');
+
+  const livebutton = document.getElementById('livebtn');
+  const sourcebutton = document.getElementById('srcbtn');
+
+  popuptitletext.innerHTML = dataobj.title;
+
+  popupdescription.innerHTML = dataobj.description;
+  popupimg.src = dataobj.mainimage;
+  livebutton.href = dataobj.livelink;
+  sourcebutton.href = dataobj.sourcelink;
+
+  document.querySelector('#modal').style.display = 'block';
+}
+function closepopup() {
+  document.querySelector('#modal').style.display = 'none';
+}
+
+function project0pop() {
+  const dataobj = proj0;
+  const popuptitletext = document.querySelector('#popuptitle');
+  const popupimg = document.querySelector('.popimg');
+  const popupdescription = document.querySelector('.popupdescription');
+
+  const livebutton = document.getElementById('livebtn');
+  const sourcebutton = document.getElementById('srcbtn');
+
+  popuptitletext.innerHTML = dataobj.title;
+
+  popupdescription.innerHTML = dataobj.description;
+  popupimg.src = dataobj.mainimage;
+  livebutton.href = dataobj.livelink;
+  sourcebutton.href = dataobj.sourcelink;
+
+  document.querySelector('#modal').style.display = 'block';
+}
+
+document.querySelectorAll('.st-see-btn_2').onclick = projectdetails(projectsarr);
+document.querySelectorAll('.closebtn').onclick = closepopup();
+document.querySelector('#st-see-btn').onclick = project0pop();
```

Comment:
- [ ] Remember to follow the Figma design to make more impressive your portfolio.
- [ ] Fix the "X" close logo to close the popup, use the right logo according to the Figma design.
- [ ] Fix the background outside your popup, it should be blurred.
- [ ] [OPTIONAL] Fix the alignment between your text and logo in your "See demo" and "See repository" buttons.

**Your design:**

![image](https://user-images.githubusercontent.com/60164239/136240965-66cceb6e-da08-4c8b-bb51-fb6ce10c7342.png)

**Figma design, design expected:**

![image](https://user-images.githubusercontent.com/60164239/136241426-fae1643b-c9b5-428b-95f6-49dc10fa6e99.png)

Reply:
This issue is well resolved! ✔️

Reply:
Thank you Robert!